backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -297,9 +299,9 @@ SearchException cannotMixElasticsearchSearchTargetWithOtherBackend(IndexSearchTa
 			value = "Index '%1$s' requires multi-tenancy but the backend does not support it in its current configuration.")
 	SearchException multiTenancyRequiredButNotSupportedByBackend(String indexName, @Param EventContext context);
 
-	@Message(id = ID_OFFSET_3 + 15,
-			value = "Unknown multi-tenancy strategy '%1$s'.")
-	SearchException unknownMultiTenancyStrategyConfiguration(String multiTenancyStrategy);
+	@Message(id = ID_OFFSET_3 + 15, value = "Invalid multi-tenancy strategy name: '%1$s'."
+			+ " Valid names are: %2$s.")
```

Comment:
maybe valid strategies?

Reply:
It looks better to me this way, but feel free to change it if you feel strongly against it.